renderdoc/os/posix/linux/linux_callstack.cpp

```diff
@@ -66,11 +66,11 @@ class LinuxCallstack : public Callstack::Stackwalk
   {
     void *addrs_ptr[ARRAY_COUNT(addrs)];
 
-    int ret = backtrace(addrs_ptr, ARRAY_COUNT(addrs));
+    size_t ret = backtrace(addrs_ptr, ARRAY_COUNT(addrs));
```

Comment:
Please don't make any changes to the linux OS files or use them in your platform's build. Instead duplicate these to freebsd_ files and make any changes there so that they are independent of any supported platforms.

Reply:
Can it be bsd_\* just like apple_\* ones? If anyone will try to run renderdoc on OpenBSD for example, there mostly aren't FreeBSD specific things.